BETA.md

```diff
@@ -1,69 +1,96 @@
-There are two alternative ways to install `semanticdb-scalac`.
 
-The first option is to enable `semanticdb-scalac` permanently for your project in `build.sbt` with:
+Some features like definition/references/hover rely on artifacts produced by a
+compiler plugin called `semanticdb-scalac`. There are two alternative ways to
+install `semanticdb-scalac`.
+
+The first option is to enable `semanticdb-scalac` permanently for your project
+in `build.sbt` with:
 
 ```scala
-libraryDependencies += compilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
+ addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
 scalacOptions += "-Yrangepos"
 ```
 
-The second option is to enable `semanticdb-scalac` only for an active sbt session by running `semanticdbEnable` from the sbt shell.
+The second option is to enable `semanticdb-scalac` only for an active sbt
+session by running `semanticdbEnable` from the sbt shell.
+
 ```scala
 $ sbt
 > semanticdbEnable // automatically runs libraryDependencies += compilerPlugin(...)
 > compile // re-compile project with semanticdb-scalac compiler plugin
 > ...
 ```
-As soon as you exit the sbt shell you need to re-run `semanticdbEnable` next time you open sbt.
+
+As soon as you exit the sbt shell you need to re-run `semanticdbEnable` next
+time you open sbt.
 
 ## Step 2 - produce the build metadata
+
 In your project of choice, open `sbt` and run `*:metalsEnableCompletions`.
-As mentioned above, this will produce the necessary metadata for the server.
+Running the task produces the necessary metadata for the server to support
+features like completions and goto definition in dependency sources.
 
-> **NOTE**: you will need to repeat this step every time you add a new source file to the project or if you run `sbt clean`
+> **NOTE**: you will need to repeat this step every time you add a new
+> dependency in your build or when you run `sbt clean`. Also,
+> the "`metalsEnableCompletions`" name is misleading since it does not
+> automatically enable completions.
 
 ## Step 3 - start editing
-Open your project in VSCode (`code .` from your terminal) and open a Scala file; the server will now start.
 
-Please note that it may take a few seconds for the server to start and there's currently no explicit
-indication that the server has started (other than features starting to work).
-To monitor the server activity, we suggest to watch the log file in your project's target directory,
-for instance: `tail -f .metals/metals.log`.
+Open your project in VSCode (`code .` from your terminal) and open a Scala file;
+the server will now start.
+
+Please note that it may take a few seconds for the server to start and there's
+currently no explicit indication that the server has started (other than
+features starting to work). To monitor the server activity, we suggest to watch
+the log file in your project's target directory, for instance:
+`tail -f .metals/metals.log`.
```

Comment:
I think it's worth mentioning that you can watch this log in the VSCode output panel (selecting Metals on the right)

Reply:
I've never done that. 

Updated.

Reply:
Good catch, I exclusively look at logs from the VSCode panel!